src/Microsoft.XmlSerializer.Generator/pkg/build/Microsoft.XmlSerializer.Generator.targets

```diff
@@ -4,14 +4,19 @@
     <_SerializerDllIntermediateFolder>$(IntermediateOutputPath)$(_SerializationAssemblyName).dll</_SerializerDllIntermediateFolder>
     <_SerializerPdbIntermediateFolder>$(IntermediateOutputPath)$(_SerializationAssemblyName).pdb</_SerializerPdbIntermediateFolder>
     <_SerializerCsIntermediateFolder>$(IntermediateOutputPath)$(_SerializationAssemblyName).cs</_SerializerCsIntermediateFolder>
-    <_SGenWarningText>SGEN : warning SGEN1: Fail to generate the serializer for $(AssemblyName).dll. Please follow the instructions at https://go.microsoft.com/fwlink/?linkid=858594 and try again.</_SGenWarningText>
   </PropertyGroup>
   <Target Name="GenerateSerializationAssembly" AfterTargets="Build">
+    <PropertyGroup>
+      <_fullAssemblyName Condition="Exists('$(IntermediateOutputPath)$(AssemblyName).dll') == 'true'">$(AssemblyName).dll</_fullAssemblyName>
```

Comment:
_fullAssemblyName -> _OutputAssemblyFullName

Can we get the file extension from output type?

Reply:
The outputtype is always exe although the assembly is end with .dll.

Reply:
I see. How about TargetExt?